nautobot/users/admin.py

```diff
@@ -198,7 +198,8 @@ def clean(self):
             for ct in object_types:
                 model = ct.model_class()
                 try:
-                    model.objects.filter(*[models.Q(**c) for c in constraints]).exists()
+                    tokens = {"$user": 0}
```

Comment:
Mind adding an explanatory comment here?

Alternately, would it maybe make sense for `qs_filter_from_constraints` to default `tokens` to `{"$user": 0}` if not otherwise specified?